config/services/dataplane_v1beta1_openstackdataplaneservice_nova.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: nova
+spec:
+  label: dataplane-deployment-nova
+  secrets:
+    - nova-cell1-compute-config
+  role:
+    name: "Deploy EDPM Nova"
+    hosts: "all"
+    strategy: "linear"
+    tasks:
+      - name: "nova"
+        import_role:
+          name: "osp.edpm.edpm_nova"
```

Comment:
this is  in https://github.com/openstack-k8s-operators/edpm-ansible/pull/217

https://github.com/openstack-k8s-operators/edpm-ansible/pull/217 is on top of 215
but i have updated 215 a few times as i have been working on it ill rebase 217